.travis.yml

```diff
@@ -3,6 +3,7 @@ language: python
 python:
   - "3.7"
   - "3.8"
+  - "3.9"
```

Comment:
The version "3.9-dev" is available for now if you'd like to use that until Travis has an official release.

Reply:
Thanks for the tip, I've updated the settings.